tests/resources/lambda_function.cpp

```diff
@@ -26,12 +27,19 @@ invocation_response binary_response(invocation_request const& /*request*/)
     return invocation_response::success(png, "image/png");
 }
 
+invocation_response crash_backtrace(invocation_request const& /*request*/)
+{
+    throw std::runtime_error("barf");
```

Comment:
Nice! having Exceptions turned on in the tests came in handy twice so far :)

Reply:
😎